src/routes/search/searchHelpers.ts

```diff
@@ -22,21 +22,22 @@ const getHighlightOrSnippetResultValue = (
   return getHighlightOrSnippetResultValue(result, depth + 1);
 };
 
-export const searchHitsTemplate: TemplateWithBindEvent<Hit<{ permalink?: string }>> = (hit) =>
-  // Note: under the post-excerpt div, replaced hit._highlightResult.content.value with
-  // hit._snippetResult.content.value, which seems more concise. Could be subject to change.
-  `
+export const searchHitsTemplate: TemplateWithBindEvent<Hit<{ url?: string }>> = (hit) => {
+  // hit._highlightResult is will only ever contain searchableAttributes that we designate
+  // Docs: https://www.algolia.com/doc/api-reference/api-parameters/searchableAttributes/
+  return `
     <div>
       <article>
         <div class="post-content">
           <h2 class="entry-title">
-            <a class="usa-link" href="${hit.permalink}" rel="bookmark">
-              ${getHighlightOrSnippetResultValue(hit._highlightResult?.post_title)}
+            <a class="usa-link" href="${hit.url}" rel="bookmark">
+              ${getHighlightOrSnippetResultValue(hit._highlightResult?.metaTitle)}
```

Comment:
I'm guessing we'll be able to remove the wrapper function and just use type assertions. I just didn't get to that yet.